api/capi/src/nnstreamer-capi-single.c

```diff
@@ -338,10 +299,8 @@ ml_single_set_gst_info (ml_single * single_h, const ml_tensors_info_h info)
     case ML_NNFW_TYPE_CUSTOM_FILTER:
       ml_tensors_info_copy_from_ml (&gst_in_info, info);
 
-      klass = g_type_class_peek (G_TYPE_TENSOR_FILTER_SINGLE);
-      if (klass == NULL
-          || klass->set_input_info (single_h->filter, &gst_in_info,
-              &gst_out_info) == FALSE) {
+      if (!single_h->klass->set_input_info (single_h->filter, &gst_in_info,
+              &gst_out_info)) {
```

Comment:
`status = single_h->klass->set_input_info (...)` ?

Reply:
return type for `set_input_info` is gboolean, and status is `ML_ERROR` type.

Reply:
Ah, right.